clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/view/EcsInstanceProvider.java

```diff
@@ -72,7 +72,6 @@ public EcsTask getInstance(String account, String region, String id) {
     List<Map<String, Object>> healthStatus =
         containerInformationService.getHealthStatus(id, serviceName, account, region);
     String address = containerInformationService.getTaskPrivateAddress(account, region, task);
-    String zone = containerInformationService.getTaskZone(account, region, task);
```

Comment:
Should this be the default if availabilityZone is not specified?

Reply:
That makes sense here. What are your thoughts on implementing that like this (similar to `networkInterface` immediately below)?

```java
String zone =
    task.getAvailabilityZone() != null
        && !task.getAvailabilityZone().isEmpty()
    ? task.getAvailabilityZone()
    : containerInformationService.getTaskZone(account, region, task);
```

Reply:
Updated!